docs/use_cases/model_applications/s2s_stratosphere/UserScript_fcstGFS_obsERA_StratospherePolar.py

```diff
@@ -1,54 +1,94 @@
 """
-Bias Plot on Polar Cap Temperature and Polar Vortex U: UserScript, Stat-Analysis
-================================================================================
+UserScript and StatAnalysis: Compute Polar Cap Temperature and Polar Vortex U and create Plots
+==============================================================================================
 
 model_applications/
```

Comment:
```suggestion
model_applications/s2s_stratosphere/UserScript_fcstGFS_obsERA_StratospherePolar.py
```
The [template](https://github.com/dtcenter/METplus/blob/develop/docs/use_cases/use_case_documentation_template.py) has the full path on one line.